supervisor/tools/ewa_tool.py

```diff
@@ -0,0 +1,47 @@
+import logging
+from pydantic import BaseModel, Field
+
+from beeai_framework.context import RunContext
+from beeai_framework.emitter import Emitter
+from beeai_framework.tools import StringToolOutput, Tool, ToolRunOptions
+
+from .testrun_analyzer import get_tcms_run_details
+
+logger = logging.getLogger(__name__)
+
+
+class EwaToolInput(BaseModel):
+    run_id: int = Field(description="TCMS Test Run ID (e.g., 12345)")
+
+
+class EwaTool(Tool[EwaToolInput, ToolRunOptions, StringToolOutput]):
+    name = "analyze_ewa_testrun"  # type: ignore
+    description = "Fetches and filters TCMS test run details by its ID."  # type: ignore
```

Comment:
Should be a bit more detailed - in particular, let's make the relationship between EWA and TCMS clear here. Something like:

```suggestion
    description = (  # type: ignore
         "Analyzes a TCMS test run generated by Errata Workflow Automation (EWA) "
         "and creates a test-case by test-case report."
    )
```